### ISSUE TITLE
2018-12-12. Paxmod 2.8-pre. Broken with Windows 7 and FF Developer 65.0b3 (64-bit)

### ISSUE THREAD

**Issue by bor8** [numirias/paxmod#27]
![image](https://user-images.githubusercontent.com/5301068/49869142-cbaedf80-fe0f-11e8-8e0d-e44017e664de.png)

**Comment by bor8** [comment]
It works halfway when I check the inline title bar. But it's not fun because it's buggier than before.

**Comment by xeonight** [comment]
Same problem in Windows 10; with the Inline Titlebar checked, the minimize, maximize, and close buttons on the right are always the full height of the tab row, including when i increase it with blank tabs.

**Comment by xeonight** [comment]
Also: i can still navigate tabs like normal using CTRL+TAB and CTRL+SHIFT+TAB, and the close tab button works too.
![image](https://user-images.githubusercontent.com/75767/49883017-0f452180-fdef-11e8-8c0e-6c35306e471c.png)

**Comment by xeonight** [comment]
Update: #26 had a solution to anyone using FF65+ (i assume), but you have to edit the 'browser.css' file inside the extension XPI, which acts as an archive, I was able to do so using 7zip, but only after disabling the addon in my running Firefox (or quitting FF).

Quote:
> 
> 
> Commenting out these titlebar attributes fixes it for FF65+.
> 
> ```
> /*#titlebar {
>     visibility: var(--paxmod-titlebar-visibility);
>     height: var(--tab-min-height) !important;
>     margin-bottom: calc(0px - var(--tab-min-height)) !important;
> }*/
> 
> /*#main-window[sizemode="maximized"] #titlebar {
>     margin-bottom:calc(var(--paxmod-titlebar-margin) - var(--tab-min-height)) !important;
> }*/
> 
> /*#main-window[sizemode="normal"] > #titlebar {
>     display: var(--paxmod-titlebar-display);
> }*/
> ```
> 
> I don't know how to do this without breaking FF64- so I'm not issuing a pull request.

Thank You B1czu

**Comment by numirias** [comment]
Seems like a duplicate of #25
